src/common/helpers/axiosClient.ts

```diff
@@ -1,17 +1,15 @@
 // #region Global Imports
+import {LANG_TYPES} from "@constants/enums";
 import axios, {AxiosInstance, AxiosRequestConfig, AxiosResponse} from "axios";
 import {Buffer} from "buffer";
 
-import {LANG_TYPES} from "../constants/enums";
 import storage from "./localStorage";
+
 const configs: AxiosRequestConfig = {
-  baseURL: "http://localhost:3000/api",
-  // baseURL: `${process.env.SERVER_API_URL}`,
+  baseURL: process.env.SERVER_API_URL || "/api",
   headers: {
     "Content-Type": "application/json",
     "Cache-Control": "no-cache",
-    "Access-Control-Allow-Origin": "*",
```

Comment:
FYI:  this header is mean to be used response header, not request header.
Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin